commit.sh

```diff
@@ -0,0 +1,3 @@
+git show --summary | awk '{print $2}' >commit.txt
```

Comment:
Why so complicated?
`git show --summary | awk '{print $2}' | head`

Reply:
Cause your result is :
59cd12962519ce61f9bb5081dc348094673ec6aa
c56e077
BasileoLuv
Fri

conflict

My:
file(newcommit.txt) with
59cd12962519ce61f9bb5081dc348094673ec6aa

Reply:
But  this is works
git show --summary | awk '{print $2}'|sed -n -e '1p' | head